dartagnan/src/test/resources/spirv/vector.spv.dis

```diff
@@ -1,6 +1,11 @@
 ; SPIR-V
 ; Version: 1.0
 ; Schema: 0
+; ================
+; Input: %v3value = Vector {0, 0, 0}
+; Output: forall (%v3v[0]=0 /\ %v3v[8]=1 /\ %v3v[16]=2)
```

Comment:
It would be nice to add a set of unit tests for different variations of valid and invalid headers.

Reply:
Again, it is the matter of taste, but I would put our annotations in the very beginning of the file before "; SPIR-V"

Reply:
Moved